[FSSDK-9434] doc: update readme with Prometheus metrics doc (#412)
optimizely/agent: README.md

### Metrics

The `/metrics` endpoint exposes telemetry data of the running Optimizely Agent.

> Currently Agent exposes two type of metrics data (expvar or prometheus) based on user's input. By default expvar metrics will be exposed. To configure this update config.yaml or
> setting the value of the environment variable `OPTIMIZELY_ADMIN_METRICSTYPE`. Supported values are `expvar` (default) & `promethues`.

Reviewer: ```suggestion
Currently, Agent exposes two type of metrics data (expvar or prometheus) based on user's input. By default, expvar metrics will be used. To configure this, update config.yaml or
set the value of the environment variable `OPTIMIZELY_ADMIN_METRICSTYPE`. Supported values are `expvar` (default) & `promethues`.
```

```yaml
##
## admin service configuration
##
admin:
    ## http listener port
    port: "8088"
    ## metrics package to use
    ## supported packages are expvar and prometheus
    ## default is expvar
    metricsType: ""
    ## metricsType: "promethues" ## for prometheus metrics
```

#### Expvar metrics

> The core runtime metrics are exposed via the go expvar package. Documentation for the various statistics can be found as part of the [mstats](https://go.dev/src/runtime/mstats.go) package.

Reviewer: ```suggestion
The core runtime metrics are exposed via the Go expvar package. Documentation for the various statistics can be found as part of the [mstats](https://go.dev/src/runtime/mstats.go) package.
```

"timers.<metric-name>.responseTimeHist.p99": 0,
```

#### Prometheus metrics

> Optimizely Agent also supports Prometheus metrics. Prometheus is an open-source systems for monitoring and alerting toolkit. You can use it to collect and visualize metrics in a time-series database.
> 

Reviewer: ```suggestion
Optimizely Agent also supports Prometheus metrics. Prometheus is an open-source toolkit for monitoring and alerting. You can use it to collect and visualize metrics in a time-series database.

```

To access the Prometheus metrics, you can use the `/metrics` endpoint with a Prometheus server. The metrics are exposed in a format that Prometheus can scrape and aggregate.

Example Request:

```bash
curl localhost:8088/metrics
```

> This will return a plain text response in the Prometheus Exposition Formats, which includes all the metrics that Prometheus is currently tracking.

Reviewer: ```suggestion
This will return a plain text response in the Prometheus Exposition Format, which includes all the metrics that Prometheus is currently tracking.
```
